src/nvim/syntax.c

```diff
@@ -5879,7 +5871,7 @@ int syn_get_foldlevel(win_T *wp, long lnum)
   if (wp->w_s->b_syn_folditems != 0
       && !wp->w_s->b_syn_error
       && !wp->w_s->b_syn_slow) {
-    syntax_start(wp, lnum);
+    syntax_start(wp, (linenr_T)lnum);
```

Comment:
This `lnum` argument is only passed a `linenr_T`, so it can be changed to a `linenr_T`.